data/updatePrereq.js

```diff
@@ -0,0 +1,342 @@
+import { connect } from "./db.js";
+import dotenv from "dotenv";
+import SISCourseV from '../model/SISCourseV.js';
+
+dotenv.config();
+
+async function updateIntroAlgo() {
+  connect();
+  const number = "EN.601.433";
+  const oldPrereq = [
+      {
+          "Description": "EN.600.226/EN.601.226 AND (EN.553.171/EN.550.171 OR EN.553.172/EN.550.170 OR EN.600.271/EN.601.231",
+          "Expression": "EN.600.226[C]^AND^(^EN.553.171[C]^OR^EN.553.172[C]^OR^EN.600.271[C]^)",
+          "IsNegative": "N"
+      },
+      {
+          "Description": "Students may receive credit for only one of EN.600.363, EN.600.463, EN.601.433, EN.601.633.",
+          "Expression": "EN.600.363[C]^OR^EN.601.633[C]",
+          "IsNegative": "Y"
+      }
+  ];
+
+  const newPrereq = [
+      {
+          "Description": "EN.600.226/EN.601.226 AND (EN.553.171/EN.550.171 OR EN.553.172/EN.550.170 OR EN.600.271/EN.601.231 OR EN.601.230",
+          "Expression": "EN.600.226[C]^AND^(^EN.553.171[C]^OR^EN.553.172[C]^OR^EN.600.271[C]^OR^EN.601.230[C]^)",
+          "IsNegative": "N"
+      },
+      {
+          "Description": "Students may receive credit for only one of EN.600.363, EN.600.463, EN.601.433, EN.601.633.",
+          "Expression": "EN.600.363[C]^OR^EN.601.633[C]",
+          "IsNegative": "Y"
+      }
+  ];
+
+  let data = await SISCourseV.find({number});
+  let course = data[0];
+  for (let version of course.versions) {
+    version.preReq = newPrereq;
+  }
+  await course.save();
+  console.log("done!")
+}
+
+// updateIntroAlgo();
+
+async function checkUpdated() {
+  connect();
+  const number = "EN.601.433";
+  let data = await SISCourseV.find({number});
+  let course = data[0];
+
+  for (let version of course.versions) {
+    console.log(version.preReq);
+  }
+}
+
+checkUpdated();
+
+
+
+/** Intro Algo
+ * {
+    "data": {
+        "_id": "635df458662f1ebee559601f",
```

Comment:
looks good, did you run this in both production and development? if not, please do so :)